manifests/crd-podgroup.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1beta1

```

Comment:
Suggest to change file name from crd-podgroup.yaml to podgroup.yaml.

Reply:
Done